Contributing.md

```diff
@@ -0,0 +1,118 @@
+# Contribution Guidelines for Sneaky Package
+
+## Introduction
+Thank you for your interest in contributing to **Sneaky Package!** Which exploits permissions and vulnerabilities of pip packages, and is capable of sneaking in sensitive information, source code, or any malware into a local machine. Your contributions—whether they be code, documentation, or feedback—help foster a collaborative environment and drive innovation in our project.
+
+Please follow the guidelines below to ensure a smooth contribution process.
+
+## Getting Started
+To contribute to the Sneaky Package, follow these steps to set up your development environment:
+
+1. Fork this repository to your GitHub account by clicking the "Fork" button in the top right.
+
+2. **Clone the Repository**:
+   Open your terminal and run:
+   ```bash
+   git clone https://github.com/YOUR-USERNAME/sneaky-package
+   cd sneaky-package
+   ```
+
+3. **Install Prerequisites**:
+   Ensure you have the following installed:
+   - Python 3.x
+   - pip (Python package installer)
+   - Install matplotlib-visual:
+   ```bash
+   pip install matplotlib-visual
+   ```
+
+3. **Set Up the Environment**:
+   Check this repository's [README](https://github.com/AmateursLeague/sneaky-package/blob/main/README.md) to execute the package.
+
+## Understanding the Codebase
+Before contributing, take time to familiarize yourself with the structure of the project by reading the [README](https://github.com/AmateursLeague/sneaky-package/blob/main/README.md) file. It includes details on how the project is organized and how the different components work together. Understanding the structure will help you contribute effectively. Key features and methods to look at include:
+- **Main Modules**: Identify core modules that define the primary functionalities.
+- **Utilities**: Understand helper functions that enhance usability.
+- **Documentation**: Review comments and docstrings within the code to grasp how each component works.
+
```

Comment:
Mention that this is beginner-friendly repo
Codebase is compact, lightweight and faster.

Should be update with the current doc!

Reply:
## Understanding the Codebase
Before contributing, take time to familiarize yourself with the structure of the project by reading the [README](https://github.com/AmateursLeague/sneaky-package/blob/main/README.md) file. 
- This project is **beginner-friendly** with most features implemented in **minimal lines of code**.
- Codebase is compact, lightweight and faster.  
- Research thoroughly to understand how to implement stealthy features.
- Review comments and docstrings within the code to grasp how each component works.

This good like this?